doc/source/api.rst

```diff
@@ -8,3 +10,12 @@ DataStructureDefinition
 
 .. autoclass:: DataStructureDefinition
    :members:
+
+
+
+
+Region processing
+-----------------
+
+.. automodule:: nomenclature.region_mapping_models
```

Comment:
```suggestion
.. autoclass:: nomenclature.region_mapping_models
```
Not sure, but I think "automodule" doesn't catch everything important. At least it looks a bit different in the build.

Reply:
Not sure what you mean by that. What is missing if you say it doesn't catch everything important?
I put automodule on purpose since I want sphinx to take all classes from the module nomenclature.region_mapping_models. If I change it to autoclass I don't get anything, which is to be expected since nomenclature.region_mapping_models is a module and not a class.
I could of course do something like this:
```rst

.. autoclass:: nomenclature.region_mapping_models.RegionProcessor
.. autoclass:: nomenclature.region_mapping_models.NativeRegion
etc...
```
But I don't think that's a good idea as there is already a command to document everything from a module.

Reply:
Ah sorry my bet, I thought it is about the RegionProcessor class since you referred to api.rst in the getting_started.rst that here are all classes listet. But since all other classes will be dealt with in a new PR it's fine as it is.